app/.gitlab-ci.yml

```diff
@@ -383,6 +383,25 @@ test:backend-format:
         - app/proto/**/*
         - app/backend/**/*
 
+test:backend-mypy:
+  needs: ["build:backend"]
+  stage: test
+  image: $BACKEND_TAG
+  inherit:
+    # no docker login
+    default: false
+  script:
+    - cd app/backend
+    - mypy src/couchers
```

Comment:
Is this mypy necessarily the same as would be selected by `uv run -m mypy`? (pinnable in pyproject.toml)